tests/validation/tests/v3_api/test_global_registry.py

```diff
@@ -0,0 +1,124 @@
+import pytest
+import errno
+import warnings
+from distutils.spawn import find_executable
+from .common import * # NOQA
+
+setup = {"p_client": None, "project": None,
+         "admin_client": None, "server_url": None,
+         "server_host": None}
+
+DISTRIBUTION_HEADER = "docker-distribution-api-version"
+REGISTRY_TEMP_ID = "catalog://?catalog=system-library&template=harbor" \
+                   "&version=1.7.5-rancher1"
+
+
+def test_global_registry_enable():
+    admin_client = setup["admin_client"]
+    p_client = setup["p_client"]
+    p = setup["project"]
+
+    setting = admin_client.by_id_setting(id="global-registry-enabled")
+    assert setting is not None, \
+        "Expected to find global-registry-enabled setting"
+
+    p_client.create_app(
+        name="global-registry",
+        externalId=REGISTRY_TEMP_ID,
+        targetNamespace="cattle-system",
+        projectId=p.id,
+        answers={
+            "expose.ingress.host": setup["server_host"],
+            "externalURL": setup["server_url"],
+            "persistence.enabled": "false",
+            "clair.enabled": "false",
+            "notary.enabled": "false"
+        },
```

Comment:
We should pass all parameters that are exposed in the UI workflow for enabling global registry in answers and the parameter values have to be ideally the default ones presented in the UI. 
This would mean , we need to have persistence.enabled , clair.enabled and notary.enabled set to True.